assets/airports/kden.json

```diff
@@ -0,0 +1,7163 @@
+                ["N40.54405", "W104.03136"],
+                ["N40.52379", "W103.99672"],
+                ["N40.46494", "W103.91126"],
+                ["N40.40361", "W103.83830"],
+                ["N40.33483", "W103.77118"],
+                ["N40.27080", "W103.71981"],
+                ["N40.18478", "W103.66781"],
+                ["N40.15425", "W103.65039"],
+                ["N40.14442", "W103.69214"],
+                ["N40.11106", "W103.82958"],
+                ["N40.01279", "W103.82769"],
+                ["N39.90051", "W103.80611"],
+                ["N39.86717", "W103.80423"],
+                ["N39.75333", "W103.81402"],
+                ["N39.61448", "W103.85948"],
+                ["N39.51308", "W103.91503"],
+                ["N39.50117", "W103.87453"],
+                ["N39.48866", "W103.83253"],
+                ["N39.47674", "W103.79203"],
+                ["N39.46423", "W103.75041"],
+                ["N39.45201", "W103.70954"],
+                ["N39.42256", "W103.73060"],
+                ["N39.37003", "W103.77257"],
+                ["N39.33774", "W103.80200"],
+                ["N39.31555", "W103.82403"],
+                ["N39.26316", "W103.88223"],
+                ["N39.24476", "W103.90492"],
+                ["N39.22321", "W103.93297"],
+                ["N39.17132", "W104.01041"],
+                ["N39.14500", "W104.05518"],
+                ["N39.13129", "W104.08079"],
+                ["N39.09570", "W104.15559"],
+                ["N39.08207", "W104.18761"],
+                ["N39.09770", "W104.19519"],
+                ["N39.13487", "W104.21398"],
+                ["N39.16612", "W104.22988"],
+                ["N39.13386", "W104.37558"],
+                ["N39.11453", "W104.52552"],
+                ["N39.10792", "W104.57815"],
+                ["N39.09888", "W104.77392"],
+                ["N39.10363", "W104.82138"],
+                ["N39.11037", "W104.86426"],
+                ["N39.10985", "W104.91184"],
+                ["N39.12967", "W105.16132"],
+                ["N39.25485", "W105.20076"],
+                ["N39.15648", "W105.32380"],
+                ["N39.13201", "W105.35419"],
+                ["N39.15182", "W105.38960"],
+                ["N39.20209", "W105.47076"],
+                ["N39.24887", "W105.53388"],
+                ["N39.31240", "W105.60605"],
+                ["N39.36682", "W105.65766"],
+                ["N39.43731", "W105.71382"],
+                ["N39.46438", "W105.73245"],
+                ["N39.46583", "W105.68863"],
+                ["N39.47153", "W105.50804"],
+                ["N39.55828", "W105.54796"],
+                ["N39.58907", "W105.56500"],
+                ["N39.70832", "W105.61286"],
+                ["N39.82350", "W105.65326"],
+                ["N39.96124", "W105.64367"],
+                ["N40.03530", "W105.60233"],
+                ["N40.19813", "W105.51627"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_ROYYL246": [39.876554, -104.523922],
```

Comment:
The naming of the invisible waypoints isn't consistent. Make sure to follow the naming convention for these:

> Any fixes that represent the intersection of a runway's inbound course and another course to a fix will be descried using the format below. Note that the runway whose approach course intersects is the one to be used, not the runway whose departure course intersects.
"_RWY12BSTER081": [25.810667, -80.322667]
Fixes may be defined based on the intersection of a runway's inbound course and an outbound radial of any fix. For a point aligned with Runway 33's approach path and the XYZ VOR's outbound radial 180, we get _RWY33XYZ180. Note that if the intersection were to be on the departure side of a given runway, the opposite runway should be used to keep with the convention of using the approach course.
"_RWY1LPIE116": [27.848198, -82.546200]

Reply:
updated